homeassistant/components/sense/__init__.py

```diff
@@ -121,9 +121,10 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
         update_interval=timedelta(seconds=300),
     )
 
-    # async_request_refresh instead of async_refresh
-    # as this can take longer than 60s
-    await trends_coordinator.async_request_refresh()
+    # This can take longer than 60s and we already know
+    # sense is online since get_discovered_device_data was
+    # successful so we do it later.
+    async_call_later(hass, 1, lambda now: trends_coordinator.async_refresh())
```

Comment:
When `async_call_later` passes your lambda to `hass.async_add_job`, it will not detect it as a coroutine function and so pass it to the executor.

Instead, you want to create a task here that is started out-of-band from HA startup. You still want to call `async_request_refresh`, as that supports bundling simultaneous requests.

```suggestion
    hass.loop.create_task(trends_coordinator.async_request_refresh())
```